test/test_suite.bats

```diff
@@ -145,14 +145,12 @@
   run bash -c "dig AAAA google.com @127.0.0.1 +short +tcp"
   printf "%s\n" "${lines[@]}"
   [[ ${lines[0]} != "::" ]]
-  [[ ${lines[1]} == "" ]]
 }
 
 @test "Known host is resolved as expected" {
-  run bash -c "dig ftl.pi-hole.net @127.0.0.1 +short"
+  run bash -c "dig A ftl.pi-hole.net @127.0.0.1 +short"
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[0]} == "139.59.170.52" ]]
-  [[ ${lines[1]} == "" ]]
+  [[ ${lines[0]} != "0.0.0.0" ]]
```

Comment:
This will be true for any value that is not 0.0.0.0? Even empty or "`;; connection timed ;`"?

Reply:
Yes, the test is not meant for successful DNS resolution but for not-blocked DNS resolution. However, we can leave the IP check for FTL as it is.

Reply:
It looks like `dig` failing completely will also end up as a green condition?

Reply:
Yes

Reply:
If that's intended then I guess it does what it says on the tin.

Reply:
You are right. I missed that. I will restore the previous test as it seems superior having this in mind.